index.html

```diff
@@ -1,17 +1,12 @@
 ---
 ---
+
 <!DOCTYPE html>
 <html lang="en">
-{% include header.html %}
-
-<body>
-    {% include navbar.html %}
-
-    {% include profile.html %}
-    
-    {% include description.html %}
-    
-    {% include projects.html %}
-</body>
+  {% include header.html %}
 
-</html>
\ No newline at end of file
+  <body>
```

Comment:
If there is no change in the file except indentation, I think this can be changed back to its previous state to avoid merge conflicts

Reply:
I've made the required change, could you please verify it